test/modules/TestResultCoordinator/Controllers/ReportController.cs

```diff
@@ -39,13 +39,8 @@ public async Task<ContentResult> GetReportsAsync()
 
             return new ContentResult
             {
-                Content = this.AddManualRunReportingHeading(JsonConvert.SerializeObject(testResultReports, Formatting.Indented)) // explicit serialization needed due to the wrapping list
```

Comment:
why do you remove this manual run reporting heading?  are you sure no one uses it?

Reply:
It was only used while we were debugging to add a message before the content. But we don't need it, and it makes it harder to parse on the other side.